uvicorn/server.py

```diff
@@ -267,8 +267,6 @@ async def shutdown(self, sockets: Optional[List[socket.socket]] = None) -> None:
         # Stop accepting new connections.
         for server in self.servers:
             server.close()
-        for sock in sockets or []:
-            sock.close()
```

Comment:
This is not needed.

[asyncio.loop.create_server](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_server) specifies:

> Note The sock argument transfers ownership of the socket to the server created. To close the socket, call the server’s [close()](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Server.close) method.

[asyncio.Server](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Server) confirms that with:

> close()(https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Server.close)
Stop serving: close listening sockets and set the [sockets](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.Server.sockets) attribute to None.